src/tests/ftest/daos_test/suite.yaml

```diff
@@ -9,7 +9,7 @@ timeout: 600
 timeouts:
   test_daos_degraded_mode: 450
   test_daos_management: 110
-  test_daos_cat_recovery: 1800
+  test_daos_cat_recovery: 3000
```

Comment:
Just pointing out this adds _**20 minutes**_ to the timeout, for a total of _**50 minutes**_. And being a `pr` test, this really slows down CI.
It's not a comment for just this test, but more so the general trend of having too many long-running `pr` tests.

Reply:
The long time for test is mainly because frequently system start/stop in every test case. So maybe remove it from **pr** tests? If necessary, it can be done in next patch.

Reply:
yes, please, otherwise it's going to be problematic.